tests/no-value-export-in-declaration-file.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { ESLintUtils } from "@typescript-eslint/utils";
+import { rule } from "../lib/rules/no-value-export-in-declaration-file";
+
+const ruleTesterDeclarationFile = new ESLintUtils.RuleTester({
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    }
+  }
+});
+
+// @ts-ignore
+ruleTesterDeclarationFile.getFilename = () => 'declaration-file.d.ts';
```

Comment:
What is the @ts-ignore for ?

Reply:
I think you can add a comment here to explain that you have to override the fileName extension in the test because : the point is to forbid exports of variables from .d.ts files, but still allow export of types and interfaces, and allow exports in .ts files